CRM/Event/Form/Participant.php

```diff
@@ -1428,7 +1428,7 @@ public function buildEventFeeForm($form) {
 
       //retrieve custom information
       $form->_values = [];
-      CRM_Event_Form_Registration::initEventFee($form, $event['id'], FALSE, $form->getDiscountID());
+      CRM_Event_Form_Registration::initEventFee($form, $event['id'], FALSE, $this->getPriceSetID());
```

Comment:
I don't think it makes any difference as things stand now, but shouldn't it be $form->getPriceSetID for consistency?

Reply:
Hmm - I've been trying to go the other way for consistency - ditch the use of form & even the passing of it

Reply:
Agreed. I guess it doesn't matter here as long as this doesn't get called from anywhere else, just looks a bit odd.

Reply:
Not sure if this is affected or not https://github.com/eileenmcnaughton/civicrm-core/blob/master/tests/phpunit/CRM/Event/Form/ParticipantTest.php#L360

Reply:
@seamuslee001 yeah that needs cleaning up - although as long as this PR passes tests it is out of scope for it